### DISCUSSION TITLE
How are levels calculated/guessed/implied?

### DISCUSSION THREAD

**Discussion by wsw70** [datalust/seq-tickets#2176] [Help]
I have just started to use seq and it looks great so far (especially compared to `less` :))

I have quite a lot of events classified as `err` and i would like to understand where the classification comes from. For instance this event

![image](https://github.com/datalust/seq-tickets/assets/1345886/97d6b527-7ca3-4786-969a-759361484832)

is obviously at level `info` and yet is is classified as `err`.

**Is there a way to fine-tune this?**

Another example is an HTTP call that comes back with a `200`, so it is OK. However:

![image](https://github.com/datalust/seq-tickets/assets/1345886/ff3e94db-7c2b-4ed3-97db-2ea807da8bcc)

**Comment by liammclennan** [comment]
Seq's ingestion format is CLEF http://clef-json.org/ which has an `@l` property, which is the level. It looks like your client is sending events with the level `err`.

**Reply by wsw70** [reply]
Thank you. Is there a way to see the raw data ingested by Seq? 
My data come from docker (a global GELF driver for all the containers) and I would like to pin-point the exact moment where the `@l` property is added (and on which basis).

I assume that there is no default on Seq side for CLEF data that would not have that property?

**Reply by liammclennan** [reply]
You could use a web debugging proxy to see the traffic. 

If no level is provided Seq defaults to 'Information'.
